tests/e2e/vmops_test.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This test suite contains tests to verify behaviors of non-default vmgroup
+
+// +build runonce
+
+package e2e
+
+import (
+	"log"
+	"fmt"
+	adminutils "github.com/vmware/docker-volume-vsphere/tests/utils/admincli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/dockercli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/inputparams"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/misc"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/ssh"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/verification"
+	. "gopkg.in/check.v1"
+)
+
+const (
+	vimCmd = "vim-cmd "
+
+	// VMId get the VM ID from ESX host
+	VMId = vimCmd + "vmsvc/getallvms|grep %s|cut -f 1 -d\" \""
```

Comment:
govmomi is failing at times and almost always on the setup in Nimbus (Photon VMs). vim-cmd is achieving the same anyway without these issues.